### PR TITLE
Moved LootLockerSDK to top level of repo removing v2 directory

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,14 +57,8 @@ It is important that in your first ever Blueprint class you set the LootLocker c
...
-Please download the sample project available [here](https://github.com/LootLocker/unreal-sdk/releases). 
-
-There you can check out our sample Blueprint examples.
+3. Fill in the info from the LootLocker dashboard (LootLockerGameKey, Platform, Game Version, Development mode and AllowTokenRefresh). You should already have an account from [https://my.lootlocker.com/login](https://my.lootlocker.com/login "https://my.lootlocker.com/login").
 
 ## Support
 
-If you have any issues or just wanna chat you can reach us on our [Discord Server](https://discord.lootlocker.io/)
\ No newline at end of file
+If you have any issues or just wanna chat you can reach us on our [Discord Server](https://discord.lootlocker.com/)
```
Comment (JohannesLoot) [inline README.md:64]: The Discord-server is still at .io actually, .com gives a 403

File: README.md [hunk 2]
```diff
@@ -57,14 +57,8 @@ It is important that in your first ever Blueprint class you set the LootLocker c
...
 1. Right click within the Event Graph of your blueprint class and locate "LootLocker Settings" category.
 2. Select "Set LootLocker Settings" in that category.
-3. Fill in the info from the LootLocker dashboard (LootLockerGameKey, Platform, Game Version, Development mode and AllowTokenRefresh). You should already have an account from [https://my.lootlocker.io/login](https://my.lootlocker.io/login "https://my.lootlocker.io/login").
-
-## Sample App
-
-Please download the sample project available [here](https://github.com/LootLocker/unreal-sdk/releases). 
-
-There you can check out our sample Blueprint examples.
+3. Fill in the info from the LootLocker dashboard (LootLockerGameKey, Platform, Game Version, Development mode and AllowTokenRefresh). You should already have an account from [https://my.lootlocker.com/login](https://my.lootlocker.com/login "https://my.lootlocker.com/login").
```
Comment (JohannesLoot) [inline README.md:60]: my.lootlocker is still my.lootlocker.io, not changed yet

Comment (kirre-bylund) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (kirre-bylund) [reply README.md]: Fixed

**Review by JohannesLoot — APPROVED** [review]
All good!

**Pull request merged**